organizations/contaazul.json

```
@@ -0,0 +1,7 @@
+{
+  "id": "CONTAAZUL",
+  "name": "ContaAzul",
```

I suspect this name should be consistent with what you had in sources?
```suggestion
  "name": "Conta Azul",
```